src/app/start-app.ts

```diff
@@ -111,8 +111,12 @@ export async function startApp(app: {domains:string[], options?:normalizedAppOpt
 			.replace('"##location##"', '#location');
 
 		server.path("/.dx", async (req) => {
-			// direct web socket connection not suppported for unyt.app, don't send .dx
-			if (!req.request.headers.get("x-forwarded-host")?.endsWith(".unyt.app")) {
+			const host = req.request.headers.get("x-forwarded-host");
+			// direct web socket connection not suppported for unyt.app with HTTP-over-DATEX, don't send .dx
+			if (!(
+				host?.endsWith(".unyt.app") // is a unyt.app subdomain
+				&& !app.hostDomains.includes(host) // can be assumed use HTTP-over-DATEX
+			)) {
```

Comment:
Maybe cleaner to use `!A || B` instead of `!(A && !B)`

Reply:
This was on purpose because I think the logical statement makes more sense with &&.
It was inverted before, but it confused me too much